src/table/context/index.ts

```diff
@@ -1,2 +1,2 @@
 export { useContextSelector } from './useContextSelector';
-export { TableContext, TableContextProvider } from './TableContext';
+export { TableContext, TableContextProvider, EMPTY_TABLE_DATA } from './TableContext';
```

Comment:
is this used somewhere?

Reply:
Yes, https://github.com/qlik-oss/sn-table/pull/1002/files#diff-ae735badc3af4f2cafbd6d1d059127069d4c4447e56a042cb4f618049493b39eL62

Reply:
as https://github.com/qlik-oss/sn-table/blob/9023f518201fbc3b9bc9db515b87ace64e7be0d8/src/table/context/TableContext.tsx#L24 tableData is set with EMPTY_TABLE_DATA